src/helpers/snapshot.ts

```diff
@@ -94,13 +94,20 @@ export async function voteValidation(
   const options: any = {};
   if (import.meta.env.VITE_SCORES_URL)
     options.url = import.meta.env.VITE_SCORES_URL;
+
+  const params = space.validation?.params || {};
+  if (space.validation.name === 'basic') {
+    params.strategies =
+      space.validation?.params?.strategies || space.strategies;
+  }
```

Comment:
Hi @samuveth It should use proposal validations and strategies, not space validations